src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -512,3 +512,15 @@ def stream_stream(
                                          request_serializer,
                                          response_deserializer, None,
                                          self._loop)
+
+
+async def channel_ready(channel: Channel) -> None:
```

Comment:
Any reason not to implement this as a method on `aio.Channel`?

Reply:
Promoted to a method of `aio.Channel`.

When I first design this API, I felt a sense of duplication, since this API can be implemented by `get_state` and `wait_for_state_change`. I'm okay with either option.

Reply:
Discussed offline. My stance here is that if the type of the first argument cannot be made broader than `Channel`, then this should just be a method of `Channel`.